content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,264 @@
+---
+reviewers:
+- gjtempleton
+- jonathan-innis
+- maciekpytel
+title: Node Autoscaling
+linkTitle: Node Autoscaling
+description: >-
+  Automatically provision and consolidate the Nodes in your cluster to adapt to demand and optimize cost.
+content_type: concept
+weight: 130
+---
+
+In order to run workloads in your cluster, you need {{< glossary_tooltip text="Nodes" term_id="node" >}}.
+Nodes in your cluster can be _autoscaled_ - dynamically
+[_provisioned_](#provisioning), or [_consolidated_](#consolidation) to provide needed capacity
+while optimizing cost. Autoscaling is performed by Node [_autoscalers_](#autoscalers).
+
+## Node provisioning {#provisioning}
+
+If there are Pods in a cluster that can't be scheduled on existing Nodes, new Nodes
+can be automatically added to the cluster - _provisioned_ - to accommodate the Pods.
+This is especially useful if the number of Pods changes over time, for example
+as a result of [combining workload and Node autoscaling](#multi-scaling).
+
+Autoscalers provision the Nodes by creating and deleting cloud provider Virtual Machines (VMs) backing them.
+
+The goal of provisioning is to make all Pods schedulable. This goal is not always attainable because of various limitations
+(for example reaching configured provisioning limits, provisioning configuration not being compatible
+with a particular set of pods, lack of cloud provider capacity).
+
+{{< note >}}
+Only the scheduling constraints of pods (for example resource requests, node selectors) are taken into account when determining Nodes to
```

Comment:
I think this statement "Only the scheduling constraints of pods... are taken into account when determining Nodes to provision." is not entirely correct for Karpenter. Maybe a better way to put it would be:

"The fundamental input for determining node provisioning is pod scheduling constraints."

This would leave open other node provisioning inputs (such as vm cost). I don't think we have to state this, but I think saying _Only_ the scheduling constraints of pods... is too strong and excludes other node autoprovisioning considerations.

cc @jonathan-innis

Reply:
I think the intent was to emphasize that real resource usage is not really considered and neither CA nor Karpenter will do anything if your pod is OOMing. I think that is worth emphasizing.
But I agree that the wording is misleading - both Karpenter and CA look at VM price (in case of CA only if you enable price expander). And I could imagine a lot of other signals - e.g. spot availability. Let's rephrase this.